src/display/api.js

```diff
@@ -281,6 +282,7 @@ function getDocument(src = {}) {
     typeof src.isOffscreenCanvasSupported === "boolean"
       ? src.isOffscreenCanvasSupported
       : !isNodeJS;
+  const isChrome = src.isChrome === true;
```

Comment:
To avoid issues, and to support this properly when running tests, can we please change it as follows?
```suggestion
  const isChrome =
    // eslint-disable-next-line no-nested-ternary
    typeof src.isChrome === "boolean"
      ? src.isChrome
      : (typeof PDFJSDev === "undefined" || !PDFJSDev.test("MOZCENTRAL")) &&
        !FeatureTest.platform.isFirefox &&
        typeof window !== "undefined" &&
        !!window?.chrome;
```